src/libs/actions/Report.js

```diff
@@ -471,52 +465,45 @@ function openPaymentDetailsPage(chatReportID, iouReportID) {
  * Marks the new report actions as read
  *
  * @param {String} reportID
- * @param {String} createdDate
  */
-function readNewestAction(reportID, createdDate) {
-    const sequenceNumber = getMaxSequenceNumber(reportID);
+function readNewestAction(reportID) {
     API.write('ReadNewestAction',
         {
             reportID,
-            createdDate,
-            sequenceNumber,
         },
         {
             optimisticData: [{
                 onyxMethod: CONST.ONYX.METHOD.MERGE,
                 key: `${ONYXKEYS.COLLECTION.REPORT}${reportID}`,
                 value: {
-                    lastReadSequenceNumber: sequenceNumber,
-                    lastReadTimestamp: Date.now(),
+                    lastReadTime: DateUtils.getDBTime(),
                 },
             }],
         });
 }
 
 /**
- * Sets the last read comment on a report
+ * Sets the last read time on a report
  *
  * @param {String} reportID
- * @param {String} createdDate
- * @param {Number} sequenceNumber
+ * @param {String} reportActionCreated
  */
-function markCommentAsUnread(reportID, createdDate, sequenceNumber) {
-    const newLastReadSequenceNumber = sequenceNumber - 1;
+function markCommentAsUnread(reportID, reportActionCreated) {
+    // We subtract 1 millisecond so that the lastReadTime is updated to just before a given reportAction's created date
+    // For example, if we want to mark a report action with ID 100 and created date '2014-04-01 16:07:02.999' unread, we set the lastReadTime to '2014-04-01 16:07:02.998'
+    // Since the report action with ID 100 will be the first with a timestamp above '2014-04-01 16:07:02.998', it's the first one that will be shown as unread
+    const lastReadTime = DateUtils.oneMillisecondBefore(reportActionCreated);
     API.write('MarkAsUnread',
         {
             reportID,
-
-            // We subtract 1 millisecond so that the lastRead is updated to just before this reportAction's created date
-            createdDate: DateUtils.getDBTime(moment.utc(createdDate).valueOf() - 1),
-            sequenceNumber,
+            isoTime: lastReadTime,
```

Comment:
Why is this sent as `isoTime` and not just `lastReadTime`? `isoTime` describes the type of timestamp but not what the variable represents. So I think it will be a little unclear from the API side?

Reply:
That is what the API call expects ([here](https://github.com/Expensify/Web-Expensify/pull/35907/files#diff-8367f109728fcc54db479ccc765a52d2115e52d35eba9c660e665be1a34411acR778)). However @chiragsalian I like Marc's suggestion of calling the API param `lastReadTime` better. What do you think.

Reply:
That is what the API call expects ([here](https://github.com/Expensify/Web-Expensify/pull/35907/files#diff-8367f109728fcc54db479ccc765a52d2115e52d35eba9c660e665be1a34411acR778)). However @chiragsalian I like Marc's suggestion of calling the API param `lastReadTime` better. What do you think.

Reply:
I don't mind either. I do like marc's suggestion. The variable is isoTime in PHP layer and in auth, updating it would be a bit tricky. I'll update web-e for now.

Reply:
PR here - https://github.com/Expensify/Web-Expensify/pull/35959